src/crypto/tools/shamirs.ts

```diff
@@ -0,0 +1,14 @@
+const shamirs = require('shamirs-secret-sharing')
+//Generate Shares
+
+
+export const _generateShards = function (data: Object, noOfShares: number, threshold: number): Buffer[] {
```

Comment:
generateShard can be moved to `lib` but create a wrapper function for `shamirSplit` and `shamirCombine` here.